downloader/src/test/kotlin/GitTest.kt

```diff
@@ -110,5 +112,24 @@ class GitTest : StringSpec() {
             val workingTree = git.getWorkingTree(zipContentDir)
             workingTree.listRemoteTags().joinToString("\n") shouldBe expectedTags.joinToString("\n")
         }
+
+        "Git correctly lists submodules" {
+            val expectedSubmodules = listOf(
+                    "analyzer/src/funTest/assets/projects/external/directories",
+                    "analyzer/src/funTest/assets/projects/external/example-python-flask",
+                    "analyzer/src/funTest/assets/projects/external/godep",
+                    "analyzer/src/funTest/assets/projects/external/jgnash",
+                    "analyzer/src/funTest/assets/projects/external/qmstr",
+                    "analyzer/src/funTest/assets/projects/external/quickcheck-state-machine",
+                    "analyzer/src/funTest/assets/projects/external/sbt-multi-project-example",
+                    "analyzer/src/funTest/assets/projects/external/spdx-tools-python",
+                    "analyzer/src/funTest/assets/projects/external/sprig"
+            ).associate { path ->
```

Comment:
Again, we might want to use `associateWith` here.

Reply:
Done.